cpp/Platform.Timestamps/CommonEraClock.h

```diff
@@ -0,0 +1,65 @@
+#pragma once
+
+#include <chrono>
+
+namespace Platform::Timestamps
+{
+
+    struct CommonEraClock
+    {
+        static constexpr uint64_t TICKS_AFTER_ANNO_DOMINI = 621355968000000000;
```

Comment:
how should we name it then?

Reply:
InCsharpStyle

Reply:
So let it be PascalCase.

Reply:
Even though constant variables in most C++ codestyles are using PascalCase, I think we should use snake_case for a clock, because we have another const variable in there: is_steady. A clock requires it to be considered as such, so there's no other way to rename it. And having two different naming conventions sitting right next to each other is kinda ugly

Reply:
Ok, we can use snake_case in this class.